core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -84,6 +84,9 @@ private TableProperties() {
   public static final String PARQUET_BATCH_SIZE = "read.parquet.vectorization.batch-size";
   public static final int PARQUET_BATCH_SIZE_DEFAULT = 5000;
 
+  public static final String ORC_VECTORIZATION_ENABLED = "read.orc.vectorization.enabled";
```

Comment:
It does not seem like we need to respect this property in any other query engine. Right, @pvary @openinx?

Reply:
There is no vectorization yet for Hive queries, so we definitely do not use it 😈
Why would this be a table property? In Hive world the engine decides if it reads the table in vectorized or in non-vectorized mode.

Thanks,
Peter